cpp/include/cudf/interop.hpp

```diff
@@ -68,5 +73,41 @@ std::unique_ptr<table> from_dlpack(
 DLManagedTensor* to_dlpack(table_view const& input,
                            rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/** @} */  // end of group
+
+/**
+ * @addtogroup interop_arrow
```

Comment:
Add this new group to `cpp/include/doxygen_groups.h`
https://github.com/rapidsai/cudf/blob/8fe2b9214a8cbed70cc4b6ee169e8d3ae201b76f/cpp/include/doxygen_groups.h#L86-L89